autoload/gina/command/blame.vim

```diff
@@ -139,16 +143,9 @@ function! s:init(args) abort
 endfunction
 
 function! s:WinLeave() abort
-  let bufname = bufname('%')
-  let scheme = gina#core#buffer#param(bufname, 'scheme')
-  let alternate = substitute(
-        \ bufname,
-        \ ':' . scheme . '\>',
-        \ ':' . (scheme ==# 'blame' ? 'show' : 'blame'),
-        \ ''
-        \)
-  if bufwinnr(alternate) > 0
-    call setbufvar(alternate, 'gina_syncbind_line', line('.'))
+  let buffers = s:blame_buffer_names()
```

Comment:
Please rename it to `bufinfo` while I feel `buffers` is a list of buffer number or names.